src/main/java/uk/gov/ons/census/casesvc/service/NewAddressReportedService.java

```diff
@@ -221,13 +231,14 @@ private Case buildCaseFromSourceCaseAndEvent(CollectionCase newCollectionCase, C
     // Set fields empty/null/blank unless they come from the event
     newCase.setOrganisationName(
         getEventValOverSource(null, newCollectionCase.getAddress().getOrganisationName()));
-    newCase.setUprn(getEventValOverSource(null, newCollectionCase.getAddress().getUprn()));
     newCase.setCeExpectedCapacity(
         getEventValOverSource(null, newCollectionCase.getCeExpectedCapacity()));
     newCase.setCaseType(
         getEventValOverSource(
             newCollectionCase.getAddress().getAddressType(), newCollectionCase.getCaseType()));
     newCase.setTreatmentCode(getEventValOverSource(null, newCollectionCase.getTreatmentCode()));
+    // If no uprn on event, a uprn will be created after case is stamped and saved with caseRef
+    newCase.setUprn(getEventValOverSource(null, newCollectionCase.getAddress().getUprn()));
```

Comment:
I don't think this is correct. All new cases will have a dummy UPRN to distinguish them from sample cases